lib/sidekiq/serializers/active_job.rb

```diff
@@ -0,0 +1,26 @@
+module Sidekiq
+  module Serializers
+    module ActiveJob
+      def self.validate(job)
+      end
+
+      def self.serialize(job)
+        job["_f"] = "aj"
+        job["args"] = ::ActiveJob::Arguments.serialize(job["args"])
+        job
+      end
+
+      def self.valid_for_deserialization?(job)
+        job["_f"] == "aj"
+      end
+
+      def self.deserialize(job)
+        job["args"] = ::ActiveJob::Arguments.deserialize(job["args"])
+        job.delete("_f")
+        job
+      end
+    end
+  end
+end
+
+Sidekiq::Serializers.register_serializer(:active_job, Sidekiq::Serializers::ActiveJob)
```

Comment:
Any reason not to put this stuff directly in S::Config?

Reply:
I wanted to build an interface that other gems could extend.

We could certainly move it into the config object but I didn't see the need at this point.

Reply:
We have a pattern where the client and server have _different_ configurations. I'm not sure it makes sense for serializers/displayers to live in those areas.